packages/jaeger-ui/src/utils/trace-id-validation.ts

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) 2020 The Jaeger Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { ValidateError } from '../types/validate-error';
+
+export function validateTraceId(traceId: string): ValidateError | null {
```

Comment:
The UI has no direct dependency on the trace ID being in any specific format. It's just an opaque string for the UI, it should not be adding any additional semantics to that string.